src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const ul = item.querySelector('ul');
+
+  if (!ul) {
+    return;
+  }
+
+  const header = document.createElement('span');
+
+  header.textContent = item.firstChild.textContent.trim();
+  item.firstChild.replaceWith(header);
```

Comment:
Replacing `item.firstChild` with `header` assumes that the first child is always a text node. If the structure of the list items changes, this could lead to errors. Ensure that `firstChild` is indeed a text node before replacing it.